TestGeneration.Extensions.NUnit/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="TestGeneration.Extensions.NUnit.113f61ed-a246-4431-809d-bcd393642488" Version="1.0.0" Language="en-US" Publisher="NUnit.Org" />
+    <Identity Id="TestGeneration.Extensions.NUnit.113f61ed-a246-4431-809d-bcd393642488" Version="1.0.5" Language="en-US" Publisher="NUnit.Org" />
```

Comment:
Actually, if you consider this a development version, the odd/even protocol
we are following would make it 1.1.5. If you are going to make an official
release as a bug fix, then 1.0.5 is more correct. From what I have
observed, I think you make an official release every time you fix
something, right?


On Tue, Mar 22, 2016 at 7:57 AM, Terje Sandstrom <notifications@github.com>
wrote:

> In TestGeneration.Extensions.NUnit/source.extension.vsixmanifest
> <https://github.com/nunit/nunit-vs-testgenerator/pull/7#discussion_r57000576>
> :
>
> > @@ -1,7 +1,7 @@
> >  <?xml version="1.0" encoding="utf-8"?>
> >  <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
> >    <Metadata>
> > -    <Identity Id="TestGeneration.Extensions.NUnit.113f61ed-a246-4431-809d-bcd393642488" Version="1.0.0" Language="en-US" Publisher="NUnit.Org" />
> > +    <Identity Id="TestGeneration.Extensions.NUnit.113f61ed-a246-4431-809d-bcd393642488" Version="1.0.5" Language="en-US" Publisher="NUnit.Org" />
>
> I just update the last number during fix, and there was 5 steps. When it
> releases it should be updated to next valid number, 1.2.0, if we continue
> the odd/even practice. Didn't want to update that number yet, would wait
> until ready for release.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit-vs-testgenerator/pull/7/files/04ba668e7b2bd06080d3d8fa727e25ab9bf31cf5#r57000576>
>